tcf_website/templates/landing/landing.html

```diff
@@ -87,15 +87,14 @@ <h4 class="card-title">Browse Courses</h4>
             </div>
 
             <div class="col-lg-4">
-                <a href="{% url 'browse' %}" class="link-unstyled">
+                <a href="{% url 'blog' %}" class="link-unstyled">
                     <div class="card mx-sm-3 my-3">
                         <div class="card-body features-icons-item mx-auto mb-2">
                             <div class="features-icons-icon d-flex">
-                                <i class="icon-chart m-auto text-tcf-indigo"></i>
+                                <i class="icon-bubble m-auto text-tcf-indigo"></i>
                             </div>
-                            <h4 class="card-title">View Grades</h4>
-                            <p class="card-text font-weight-light mb-0">Check out actual grade distributions from past
-                                semesters</p>
+                            <h4 class="card-title">View Blog</h4>
+                            <p class="card-text font-weight-light mb-0">Check out what's going on with your favorite classes from our community</p>
```

Comment:
```suggestion
                            <p class="card-text font-weight-light mb-0">Join our community and check out what's going on with your favorite classes</p>
```
Might want to discuss this copy some more, but here's a suggestion